src/ResourceManager/KeyVault/Commands.KeyVault/help/Add-AzKeyVaultManagedStorageAccount.md

```diff
@@ -21,7 +21,7 @@ Add-AzKeyVaultManagedStorageAccount [-VaultName] <String> [-AccountName] <String
 ## DESCRIPTION
 Sets up an existing Azure Storage Account with Key Vault for Storage Account keys to be managed by
 Key Vault. The Storage Account must already exist. The Storage Keys are never exposed to caller.
-Key Vault auto regenerates and switches the active key based on the regeneration period.
+Key Vault auto regenerates and switches the active key based on the regeneration period. See [Azure Key Vault managed storage account - PowerShell](/azure/key-vault/key-vault-overview-storage-keys-powershell) for an overview of this feature.
```

Comment:
@BryanLa would you mind replacing the relative URL that you provided with the full URL?

Reply:
Sure, I can do that, but we typically let OPS construct the full URL. The version I'm using is relative to the site (docs.microsoft.com), so it won't change. Even if it does, our best practice is to set up redirects whenever we move/rename anything. LMK if you still prefer, and I will update my branch...

Reply:
Since this link could also be accessed by users in our repository and from the `Get-Help` cmdlet in PowerShell, I think it would be best to use the full URL. If you'd like you can create an aka.ms link and use that rather than the full URL if you'd like to make sure it's quickly updated whenever the site changes.